queue/src/fiskaltrust.Middleware.Storage.Base/BaseStorageBootStrapper.cs

```diff
@@ -66,6 +68,44 @@ public StorageBaseInitConfiguration ParseStorageConfiguration(Dictionary<string,
             }
         }
 
+        public enum Migrations
+        {
+            JournalFRCopyPayload
+        }
+
+        public async Task PersistMigrationDataAsync(IJournalFRCopyPayloadRepository journalFRCopyPayloadRepository, IMiddlewareJournalFRRepository journalFRRepository)
+        {
+            if (await journalFRCopyPayloadRepository.HasEntriesAsync().ConfigureAwait(false) == false)
+            {
+                await PerformMigrationInitialization(new List<Migrations> { Migrations.JournalFRCopyPayload }, journalFRCopyPayloadRepository, journalFRRepository).ConfigureAwait(false);
+            }
+        }
+
+        public async Task PerformMigrationInitialization(IEnumerable<Migrations> migrations, IJournalFRCopyPayloadRepository journalFRCopyPayloadRepository, IMiddlewareJournalFRRepository journalFRRepository)
+        {
+            foreach (var migration in migrations)
+            {
+                switch (migration)
+                {
+                    case Migrations.JournalFRCopyPayload:
+                        await PopulateFtJournalFRCopyPayloadTableAsync(journalFRCopyPayloadRepository, journalFRRepository);
+                        break;
+                }
+            }
+        }
+
+        private async Task PopulateFtJournalFRCopyPayloadTableAsync(IJournalFRCopyPayloadRepository journalFRCopyPayloadRepository, IMiddlewareJournalFRRepository journalFRRepository)
```

Comment:
We need unit tests for the `PopulateFtJournalFRCopyPayloadTableAsync` method.

Reply:
Already created and added [here](https://github.com/fiskaltrust/middleware/pull/199/commits/000c0abd255b44b9f162e9aec9429b80d21fa070)